tests/Oryx.Integration.Tests/StorageAccountSanityTestBase.cs

```diff
@@ -45,127 +45,109 @@ public abstract class StorageAccountSanityTestBase
             _sdkStorageAccountAccessToken = token;
         }
 
-        [Fact(Skip = "Skipping test temporarily till we resolve the issue #1663008")]
```

Comment:
nit: what are the changes in this file for?

Reply:
As we had a bit of a time crunch to get the binaries out to partners, we ignored these tests in order to disable them because we had added a version (7.0.100) to the versionToBuild.txt file that never made it to the DEV or PROD storage accounts, as those are publicly accessible. That meant that these tests would fail for the nightly and CI jobs. So we skipped them and made a [ticket](https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1663008) to address the case fully. 

I'm removing the skip here, since we should be able to get around to addressing that ticket before the next release of private binaries (we don't even know when that will be).

A quick 1-2 line alternative I could implement here is to configure the test to ignore any lines in the versionToBuild.txt that supply a download url (or even a download url that contains "dotnet-private"). But if we want to think about it some more, then I was just going to leave it for the ticket to handle.

Reply:
OH yeah good catch, thanks!